traffic_ops/client/federation_resolver.go

```diff
@@ -0,0 +1,116 @@
+package client
+
+/*
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+import "encoding/json"
+import "fmt"
+import "net/http"
+import "net/url"
+import "strconv"
```

Comment:
All files in the `client` package use the following import style. Consider matching for consistency.

```go
import (
  "a"
  "b"
  ...
)
```

Reply:
That's not true. invalidationjobs and traffic_stats use the same style - as well as several others outside of the `client` package. and we don't have a style guide - only `go fmt` is enforced (which doesn't care).